packages/appeals-service-api/src/lib/config.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
       path: process.env.DOCS_API_PATH || path.join(__dirname, '..', '..', 'api'),
     },
   },
+  documents: {
+    timeout: Number(process.env.DOCUMENTS_SERVICE_API_TIMEOUT || 10000),
```

Comment:
Personally I think this is more prone to error than explicitly parsing:

```
timeout: parseInt(process.env.DOCUMENTS_SERVICE_API_TIMEOUT, 10) || 10000
```

![Screenshot from 2021-03-15 15-29-39](https://user-images.githubusercontent.com/6329541/111178668-4a70c500-85a3-11eb-8547-fdaf4edf8d02.png)

Reply:
Yeah, you're probably right. The `Number(process.env.ENVVAR || 'default')` is what we've done everywhere else. It doesn't make it right, but it's at least consistent. I use the `Number` because it does tend fail faster if you put the wrong value in, which can be advantageous (it can also be a pain in the backside).

Reply:
Just personal preference. I'm nothing if not opinionated ;)

Reply:
OK